examples/db-templates/mongodb-ephemeral-template.json

```diff
@@ -16,7 +16,8 @@
   },
   "message": "The following service(s) have been created in your project: ${DATABASE_SERVICE_NAME}.\n\n       Username: ${MONGODB_USER}\n       Password: ${MONGODB_PASSWORD}\n  Database Name: ${MONGODB_DATABASE}\n Connection URL: mongodb://${MONGODB_USER}:${MONGODB_PASSWORD}@${DATABASE_SERVICE_NAME}/${MONGODB_DATABASE}\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/sclorg/mongodb-container/blob/master/3.2/README.md.",
   "labels": {
-    "template": "mongodb-ephemeral-template"
+    "template": "mongodb-ephemeral-template",
+    "app": "mongodb-ephemeral-template"
```

Comment:
The app label should be parameterized so that when a DB template is used with another application and we want to associate it with that app we can.